op-node/rollup/derive/attributes_queue_test.go

```diff
@@ -47,7 +49,21 @@ func TestAttributesQueue(t *testing.T) {
 		Transactions: []eth.Data{eth.Data("foobar"), eth.Data("example")},
 	}}
 
-	l1InfoTx, err := L1InfoDepositBytes(safeHead.SequenceNumber+1, l1Info)
+	parentL1Cfg := eth.SystemConfig{
+		BatcherAddr: common.Address{42},
+		Overhead:    [32]byte{},
```

Comment:
Is the main benefit here of keeping the overhead and scalar as `[32]byte` not needing to deserialize it? I suppose its never used in any arithmetic here so no need to change into a `big.Int`

Reply:
It's never used for arithmetic indeed, so it avoids redundant error edge cases by just not deserializing it.